src/redux/reducers/siloDomainData/editEntries.js

```diff
@@ -420,6 +434,89 @@ const setEntryCommentsCount = (state, action) => {
     return newState;
 };
 
+const setEntriesVerificationStatus = (state, action) => {
+    const {
+        leadId,
+        entries: entriesFromServer,
+    } = action;
+
+    const {
+        editEntries: {
+            [leadId]: {
+                entries = [],
+            } = {},
+        } = {},
+    } = state;
+
+    const newState = produce(state, (safeState) => {
+        if (!safeState.editEntries) {
+            // eslint-disable-next-line no-param-reassign
+            safeState.editEntries = {};
+        }
+        if (!safeState.editEntries[leadId]) {
+            // eslint-disable-next-line no-param-reassign
+            safeState.editEntries[leadId] = {};
+        }
+        if (!safeState.editEntries[leadId].entries) {
+            // eslint-disable-next-line no-param-reassign
+            safeState.editEntries[leadId].entries = [];
+        }
+        entriesFromServer.forEach((es) => {
+            const index = entries.findIndex(e => es.id === entryAccessor.serverId(e));
+
+            if (index > -1) {
+                const safeEntry = safeState.editEntries[leadId].entries[index];
+                // eslint-disable-next-line no-param-reassign
+                safeEntry.serverData.verified = es.verified;
+            }
+        });
+    });
+
+    return newState;
+};
+
+const setEntryVerificationStatus = (state, action) => {
+    const {
+        leadId,
+        entry,
+    } = action;
+
+    const {
+        editEntries: {
+            [leadId]: {
+                entries = [],
+            } = {},
+        } = {},
+    } = state;
+
+    const newState = produce(state, (safeState) => {
+        if (!safeState.editEntries) {
+            // eslint-disable-next-line no-param-reassign
+            safeState.editEntries = {};
+        }
+        if (!safeState.editEntries[leadId]) {
+            // eslint-disable-next-line no-param-reassign
+            safeState.editEntries[leadId] = {};
+        }
+        if (!safeState.editEntries[leadId].entries) {
+            // eslint-disable-next-line no-param-reassign
+            safeState.editEntries[leadId].entries = [];
+        }
+        const index = entries.findIndex(e => entry.entryId === entryAccessor.serverId(e));
+        if (index > -1) {
+            // eslint-disable-next-line no-param-reassign
+            safeState.editEntries[leadId].entries[index]
+                .serverData.versionId = entry.versionId;
```

Comment:
I think it's best to not update the versionId. It will only silently increase  the version without updating the real value associated with that version.

Reply:
I think its fine as we reject the verification if the version is mismatched.

Reply:
Validation is a metadata so it should not be concerned with the version id.